mlflow/server/js/src/common/utils/ActionUtils.js

```diff
@@ -130,6 +130,28 @@ export class ErrorWrapper {
     }
     return 'INTERNAL_SERVER_ERROR';
   }
+
+  // Does a best-effort at rendering an arbitrary HTTP error. If it's a DatabricksServiceException,
+  // will render the error code and message. If it's HTML, we'll strip the tags.
+  renderHttpError() {
+    const responseText = this.xhr.responseText;
+    if (responseText) {
+      try {
+        const parsed = JSON.parse(responseText);
+        if (parsed.error_code && parsed.message) {
+          return parsed.error_code + ': ' + parsed.message;
+        }
+      } catch (e) {
+        // Do our best to clean up and return the error: remove any tags, and reduce duplicate
+        // newlines.
+        let simplifiedText = responseText.replace(/<[^>]+>/gi, '');
+        simplifiedText = simplifiedText.replace(/\n\n+/gi, '\n');
+        simplifiedText = simplifiedText.trim();
+        return simplifiedText;
```

Comment:
```suggestion
        return responseText
          .replace(/<[^>]+>/gi, '')
          .replace(/\n\n+/gi, '\n')
          .trim();
```

Can we just chain methods here?

Reply:
Agreed this would be good to do, but I think it's best to handle this kind of refactor in a follow-up PR (this PR has gotten a bit complicated)